### ISSUE TITLE
Squirrel

### ISSUE THREAD

**Issue by whichonespink44** [whichonespink44/Appalachia#157]
**Characteristics:**
* Drops Raw Squirrel (can be smelted for Cooked Squirrel) and/or Nuts/Acorns (which can be used to breed Deer)
* Should be able to climb up and down trees.
* Should make sounds, especially when running through leaves of any kind... as such, probably doesn't need an ambient/idle sound.

**Comment by whichonespink44** [comment]
@dexman545 from Discord:

```
    Model & Animation
        walking
        sitting up
        big-cheek version
            full mouth of nuts
        Ears move
    AI
        climb trees
        will follow player for food
        will sit up and look at player for food
        will go up trees at night
        can survive from high falls
            IRL they can survive drops from 100 ft
        can go fast
            IRL they move at 20 mph
        will flee player and monsters
    Sounds
        passive sounds
        angry chirping
        eating sounds
```

**Comments:**
All sounds good, except...
* Not sure it needs passive sounds, but not totally against the idea
* Not sure it should follow the player, unless it's tamed

**Comment by garantiertnicht** [comment]
Is the AI in that list ordered?

**Comment by dexman545** [comment]
@garantiertnicht not really...I just through down ideas. And the following of the player was if it were tamed. As in, you hold acorns in hand and are still, then it will come to you, but if you want it to follow you you will have to feed it some
